tests/python/pants_test/backend/python/tasks/test_pytest_run.py

```diff
@@ -52,6 +52,16 @@ def task_type(cls):
         PytestPrepCoverageVersionPinned,
     ]
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.resolver_cache_dir = safe_mkdtemp()
+
+    def set_other_options(self):
```

Comment:
Consider renaming this something more descriptive like `set_resolver_cache_dir`.

Reply:
I'm not sure that's better since it makes it sound like the detail is important when it in fact isn't to the two callers. They'll always want to set other fixed options, whatever the set are, in addition to the user supplied ones. Those callers themselves don't actually care what they are at all.